packages/core/content-type-builder/admin/src/translations/en.json

```diff
@@ -60,7 +60,7 @@
   "error.contentTypeName.reserved-name": "This name cannot be used in your project as it might break other functionalities",
   "error.validation.enum-duplicate": "Duplicate values are not allowed",
   "error.validation.enum-empty-string": "Empty strings are not allowed",
```

Comment:
`Empty values are not allowed (special characters are not counted)` ?

Reply:
Discussed above.